esrally/mechanic/supplier.py

```diff
@@ -509,7 +509,7 @@ def __init__(self, repo, version, distributions_root):
 
     def fetch(self):
         io.ensure_dir(self.distributions_root)
-        download_url = self.repo.download_url
+        download_url = net.add_url_param(self.repo.download_url, {"x-elastic-no-kpi": "true"})
```

Comment:
Having URL parameter specifics in this file (which deals with matter in a higher layer of abstraction) feels a bit out of place to me.

I wonder, would it make sense to just have an additional convenience function in `net.py` like `net.url_elastic_no_kpi()` where this param is added by invoking `net.add_url_param()`?

Reply:
makes sense, i will add that